scripts/docker-helper-4.x/cht-docker-compose.sh

```diff
@@ -261,23 +262,45 @@ get_system_and_docker_info(){
   echo "---DEBUG INFO---"
   echo "Load: $(get_load_avg)"
   echo "CHT Containers: $(get_running_container_count)"
-  echo "Global Containers $(get_global_running_container_count)"
+  echo "Global Containers: $(get_global_running_container_count)"
+  echo "URL: $projectURL"
   echo
   echo $"$info" | column -t
 }
 
+update_nginx_local_ip_tls_cert(){
+  nginxContainerId=$1
+  rm -f /tmp/local-ip-fullchain /tmp/local-ip-key
+  curl  --retry 3 --fail --silent --show-error -o /tmp/local-ip-fullchain https://local-ip.medicmobile.org/fullchain
+  curl  --retry 3 --fail --silent --show-error -o /tmp/local-ip-key https://local-ip.medicmobile.org/key
+  docker cp /tmp/local-ip-fullchain "${nginxContainerId}":/etc/nginx/private/cert.pem  1>/dev/null
+  docker cp /tmp/local-ip-key "${nginxContainerId}":/etc/nginx/private/key.pem  1>/dev/null
+  docker exec "$nginxContainerId" bash -c "nginx -s reload"  2>/dev/null
+}
+
+validate_tls(){
+  url=$1
+  # by default curl validates TLS.  If we get back  60 then TLS isn't valid:
+  # exitcode: https://everything.curl.dev/usingcurl/verbose/writeout.html
+  # 60: https://everything.curl.dev/cmdline/exitcode.html
+  status=$(curl   --retry 3 --write-out "%{exitcode}"  -qs  "$url" -o /dev/null)
```

Comment:
It looks like `--retry` doesn't actually retry in some cases :disappointed: I'm still running into the `Failed to install...` error

While spinning up a CHT instance with docker helper, I ran this command over and over in a different terminal:
```sh
curl --retry 3 --write-out "%{exitcode} %{num_retries}" --fail -qs https://192-168-1-29.local-ip.medicmobile.org:10443 -o /dev/null
```
It prints the exit code and the number of retries performed. For the exit codes 7 and 60 (respectively "Failed to connect to host" and "Peer certificate cannot be authenticated"), curl doesn't retry the request. At one point between the last exit code `60` and the first exit code `0` it did retry twice

![image](https://github.com/user-attachments/assets/53e846c1-90aa-4846-a07e-650659813a53)

Reply:
ah - great sleuthing! I think we can use your manual retry method below.  thank you!